pbj-core/pbj-grpc-helidon/src/main/java/com/hedera/pbj/grpc/helidon/PbjProtocolHandler.java

```diff
@@ -300,6 +310,7 @@ public Http2StreamState streamState() {
     @Override
     public void rstStream(@NonNull final Http2RstStream rstStream) {
         incoming.onComplete();
+        outgoing.onComplete();
```

Comment:
No, I just assumed this would be called if an rst stream frame is received.

Do you have a test procedure that causes an rst stream to be sent by the client?